pkg/oam/trait/doc.go

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2019 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package trait
```

Comment:
Nit: Does this directory need to be a package?

Reply:
I think so, there could be a number of shared modifiers here. Do you think no?

Reply:
I was more wondering whether there was a technical reason to make this a Go package at the moment, given it contains no code. It seems like we're making it a package now because we may add code to it in future, rather than because we actually need it to be a package now. I'm fine with that; it doesn't hurt anything.